Kickstarter-iOS/Views/Controllers/RootTabBarViewController.swift

```diff
@@ -95,10 +95,13 @@ public final class RootTabBarViewController: UITabBarController {
   public func switchToMessageThread(_ messageThread: MessageThread) {
     self.switchToProfile()
 
+    let isDash = AppEnvironment.current.config?.features["ios_backer_dashboard"] == .some(true)
+
     guard let profileNav = self.selectedViewController as? UINavigationController,
-      let profileVC = profileNav.viewControllers.first as? ProfileViewController else {
-        return
-    }
+      let profileVC = (isDash
+        ? profileNav.viewControllers.first as? BackerDashboardViewController
+        : profileNav.viewControllers.first as? ProfileViewController)
```

Comment:
@theginbin but it doesnt look like we use that type info at all. we just stuff the `profileVC` into the `viewControllers` array.